.github/workflows/github-actions.yaml

```diff
@@ -41,5 +32,5 @@ jobs:
       - name: Grant execute permission for gradlew
         run: chmod +x gradlew
 
-      - name: Compile and run checks
-        run: ./gradlew build
+      - name: Compile
+        run: ./gradlew assemble
```

Comment:
why did you switch to assemble task>

Reply:
Some issue with slf4. In tests gives this error :

SLF4J: Class path contains multiple SLF4J providers.
SLF4J: Found provider [org.slf4j.reload4j.Reload4jServiceProvider@38ba6ce3]
SLF4J: Found provider [ch.qos.logback.classic.spi.LogbackServiceProvider@d278d2b]
SLF4J: See https://www.slf4j.org/codes.html#multiple_bindings for an explanation.
SLF4J: Actual provider is of type [org.slf4j.reload4j.Reload4jServiceProvider@38ba6ce3]

Working on this. Almost done

Reply:
Done